### PR TITLE
Hacklanding fixes

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

Can check the result here https://octavio.stage.gitcoin.co/hackathon/ethhack2019
##### Checklist

- [ ] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/assets/v2/css/ethhack2019.css [hunk 1]
```diff
@@ -4,98 +4,41 @@ a {
...
-    padding-bottom: 8rem !important;
-  }
-  .bottomnav {
-    padding: 5rem !important;
-  }
+h1,
+h2,
+h3,
+h4 {
+  font-family: Arial-Black, Helvetica, sans-serif;
```
Comment (thelostone-mc) [inline app/assets/v2/css/ethhack2019.css]: new font-family ? I thought we were using Muli throughout the site :P

File: app/assets/v2/css/ethhack2019.css [hunk 2]
```diff
@@ -4,98 +4,41 @@ a {
...
-    font-size: 1rem !important
-  }
-  .topnav {
-    font-size: 1em !important
-  }
-  .part {
-    font-size: 3rem !important
-  }
+body {
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (thelostone-mc) [inline app/assets/v2/css/ethhack2019.css]: Muli ?

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 3]
```diff
@@ -5,64 +5,67 @@
...
-          <img class="img-fluid" style="width: 100%" src="{% static "v2/images/hackathon/microsoft.svg" %}" />
-        </div>
-        <div class="col-3">
-          <img class="img-fluid" style="width: 100%" src="{% static "v2/images/hackathon/ethmain.svg" %}" />
-        </div>
+    <footer style="background: black" class="pt-5">
+      <div class="d-flex align-items-center justify-content-around p-5 flex-column flex-md-row">
+        <a target="_blank" href="https://gitcoin.co/"><img class="img-fluid m-md-5 my-5" width="295" height="68" src="{% static "v2/images/top-bar/gitcoin-logo.svg" %}" /></a>
+        <a target="_blank" href="https://www.microsoft.com"><img class="img-fluid m-md-5 my-5" width="206" height="44" src="{% static "v2/images/hackathon/microsoft.svg" %}" /></a>
+        <a target="_blank" href="https://etherealsummit.com/"><img class="img-fluid m-md-5 my-5" width="214" height="77" src="{% static "v2/images/hackathon/ethmain.svg" %}" /></a>
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: can we add `rel="noopener noreferrer" ` to the above 3 link  tags ?

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 4]
```diff
@@ -5,64 +5,67 @@
...
-      <h2 class="font-weight-bolder part" style="color: black; padding-top: 3rem; font-size: 1.2rem; text-decoration: underline; text-underline-position: under">Register as a Hacker</h2>
-      <h2 class="pdesc" style="color: black; padding-top: 1.25rem; padding-right: 3rem; padding-left: 3rem; font-size: 0.6rem">Register by April 15th for updates on key dates, prizes, teams, and instructions on how to start bounty hunting.</h2>
-      <a href="https://gitcoin.typeform.com/to/j7CSbV" class="btn font-weight-bold pbtn" style="background-color: #f9006c; border-radius: 30px; width: 12rem; font-size: 1rem; margin-top: 2rem; margin-bottom: 3rem">Register</a>
+    <section class="bg-white p-md-5 py-5">
+      <div class="row justify-content-center p-md-5 p-2">
+        <div class="col-12 col-md-9">
+          <h2 class="font-weight-bolder title-line mt-5">Register as a Hacker</h2>
+          <p class=" m-md-5 mt-5">Register by April 15th for updates on key dates, prizes, teams, and instructions on how to start bounty hunting.</p>
+          <a target="_blank" href="https://gitcoin.typeform.com/to/j7CSbV" class="btn btn-hackathon btn-circle btn-gc-pink">Register</a>
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: `rel="noopener noreferrer" `

File: app/dashboard/templates/dashboard/hackathon/ethhack_2019.html [hunk 5]
```diff
@@ -5,64 +5,67 @@
...
+          <h1 class="font-weight-bolder my-5">Ethereal Virtual Hackathon</h1>
+          <p class="font-weight-bold" style="opacity: .7;font-family: arial-black;">Powered by</p>
+          <div class="d-flex justify-content-center align-items-center flex-column flex-md-row">
+            <img class="img-fluid" width="270" height="67" style="" src="{% static "v2/images/top-bar/gitcoin-logo.svg" %}">
+            <i class="fa-plus fas mx-5 mb-0 font-bigger-3"></i>
+            <img class=" img-fluid" height="67" width="313" style=" margin-top: 0.2rem" src="{% static "v2/images/hackathon/microsoft.svg" %}" />
+          </div>
+          <div class="m-5">
+              <a target="_blank" href="https://gitcoin.typeform.com/to/j7CSbV" class="btn btn-hackathon btn-circle btn-gc-pink m-3">Register</a>
+              <a target="_blank" href="https://gitcoin.typeform.com/to/YeIflB" class="btn btn-hackathon btn-circle btn-gc-blue m-3">Fund a Prize</a>
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/hackathon/ethhack_2019.html]: rel="noopener noreferrer" for the above 2 ^

File: app/assets/v2/css/ethhack2019.css (hunk 1 above)
Comment (octavioamu) [reply app/assets/v2/css/ethhack2019.css]: not in this design, is something separated, also without the top navbar and footer

**Pull request merged**